sample/build.gradle

```
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
   compileSdkVersion rootProject.ext.compileSdkVersion
-  buildToolsVersion rootProject.ext.buildToolsVersion
+  buildToolsVersion '28.0.3'
```

revert back to rootProject.ext.buildToolsVersion
```suggestion
  buildToolsVersion rootProject.ext.buildToolsVersion
```

and change `buildToolsVersion = '27.0.3'` to `buildToolsVersion = '28.0.3'` in logger/build.gradle instead